test/xkeyboard-config-test.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python3
```

Comment:
Let's use `#!/usr/bin/env python3`.

BTW, from cursory look I think the only python3-only thing used is `subprocess.run`. So it might be worthwhile to switch to `subprocess.call` or such and then use just `/usr/bin/env python`.